macroid-core/src/main/scala/macroid/Contexts.scala

```diff
@@ -45,12 +46,19 @@ object ContextWrapper {
 
   def apply[F](fragment: F)(implicit fragmentImpl: Fragment[F]): ActivityContextWrapper =
     ContextWrapper(fragmentImpl.activity(fragment))
+
+  def apply(view: View): ContextWrapper = view.getContext match {
+    case activity: Activity => ContextWrapper(activity)
+    case service: Service => ContextWrapper(service)
+    case app: Application => ContextWrapper(app)
+  }
+
 }
 
 /** FragmentManager context, used to manipulate fragments within an Activity or another Fragment
   */
 @implicitNotFound("""Could not find `FragmentManagerContext[${F}, ${M}]`.
-If you are inside Activity or Fragment, extend Contexts[Activity] or Contexts[Fragment],
+If you are inside Activity or Fragment, extend Contexts[Activity] Contexts[Fragment], Contexts[Service] or Contexts[View],

```

Comment:
I think this change is not correct, since there no `FragmentManagerContext` for `View` was introduced.